setup.py

```diff
@@ -56,7 +56,7 @@
 EXTRAS_REQUIRE = {
     'doc': [
         'sphinx==3.2',
-        'nbsphinx==0.8.5',
+        'nbsphinx==0.8.8',
```

Comment:
This resolves an unrelated incompatibility between `jinja2` and `markupsafe` that was causing the docs build to fail.